Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 gem 'jquery-rails'
 gem 'simple_form'
 
-gem 'devise', '4.7.3'
+gem 'devise', '4.8.1'
```

Comment:
Most recent devise is 4.9.2. Why this (older) version?

Reply:
Good point. I just chose the first release that would fix the particular bug I was running into. I'll update it and see what happens.